lib/mainscreen.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'Seconscreen.dart';
+
+class MainScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("My App"),
+        centerTitle: true,
+        backgroundColor: Colors.orangeAccent,
+        automaticallyImplyLeading: false,
+      ),
+      body: Container(
+        alignment: Alignment.center,
+        padding: EdgeInsets.only(top: 16.0),
+        margin: EdgeInsets.all(100.0),
```

Comment:
Please read the docs carefully. The margin should be 32, and it should be applied only to left and right. Here margin is applied to all.